async & performance/ch1.md

```diff
@@ -57,7 +57,7 @@ function now() {
 }
 
 function later() {
-  a = a * 2;
+  answer = answer * 2;
```

Comment:
Para entendimento, acredito que não há necessidade de edição.
Atenciosamente

Reply:
Precisa sim. Neste _snippet_ não existe a variável `a`, em nenhum escopo. A variável é `answer` mesmo. Foi uma [modificação feita pelo autor](https://github.com/getify/You-Dont-Know-JS/commit/8b5052c9d6d5ed9c630f9a36f44d906c252a983a) no livro original, que não foi trocada nesta função.

Reply:
oi @victorpothin esse é um dos requerimentos do autor do livro para essa autorizar a tradução. Os blocos de código, salvo comentários, não podem ser alterados.